guides/source/api_app.md

```diff
@@ -151,6 +151,113 @@ This will do three main things for you:
 - Configure the generators to skip generating views, helpers, and assets when
   you generate a new resource.
 
+### Generating a New Resource
+
+To see how our newly created API handles generating a new resource, let's create
+new Group and Member resources. Each group will have a name, and each member will
+have an associated group_id and name column.
+
+```bash
+$ rails g scaffold Group name:string
+$ rails g model Member name:string group:references
+```
+
+Before we can use our scaffolded code, we need to update our database scheme.
+
+```bash
+$ rails db:migrate
+```
+Our models should be set up as:
+```ruby
+# app/models/group.rb
+class Group < ApplicationRecord
+  has_many :members
+end
+```
+```ruby
+# app/models/member.rb
+class Member < ApplicationRecord
+  belongs_to :group
+end
+```
+
+Now if we open our Groups Controller, we should notice that with an API rails
+app we are rendering JSON data only. On the index action we query for Group.all
+and assign it an instance variable called @groups. That will automatically return
+a JSON object containing groups.
```

Comment:
@groups doesn't return a JSON object. It gets turned into JSON by the render call.
Maybe something like:
```suggestion
and assign it an instance variable called `@groups`. Passing it to `render` with the
`:json` option will automatically render the groups as JSON.
```